TCAT/Views/LiveIndicator.swift

```diff
@@ -24,39 +29,54 @@ class LiveIndicator: UIView {
     fileprivate let DIM_OPACITY: CGFloat = 0.5
     fileprivate let INTERVAL: TimeInterval = LiveIndicator.INTERVAL
     
+    /// The color to draw the indicator with
+    var color: UIColor = .white
```

Comment:
not a huge fan of default values unless they properly describe a default way of initializing (like if you have an `init()`). Think its better if you initialize color before the super.init() calls of both inits, rather than provide a random default value.

Reply:
It's not random, it really is the default for the LiveIndicator. Do you think it should be a default parameter in the init instead?

Reply:
I removed the default by the variable, left it in parameters